README.md

```diff
@@ -109,6 +109,12 @@ To run tests, run the following command accordingly:
 - Twitter: [@twitterhandle](https://twitter.com/@monzo200)
 - LinkedIn: [LinkedIn](https://https://www.linkedin.com/in/felixnyamai/)
 
+👤 **Ajayi Ridwan**
+
+- GitHub: [@githubhandle](https://github.com/Ridwanullahi-code)
+- Twitter: [@twitterhandle](https://twitter.com/Ridwanullahi22)
+- LinkedIn: [LinkedIn](https://www.linkedin.com/in/ajayi-ridwan-2a1515199/)
+
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
 <!-- FUTURE FEATURES -->
```

Comment:
- [ ] Kindly make your readme contents look similar as you can see in the microverse readme template to make your readme look clean, professional, and descriptive 😊 because your readme contents are in the center which is making your readme not look professional 😨

Reply:
Thank you for your observation...I have updated